### PR TITLE
Active storage feature

### PR DESCRIPTION
**Pull request by Mildred14**
Description: I added mini_magick gem and his configurations.
What implements/changes?: Just implemented mini_magick gem
How do you manually test this?: -
Before - After screenshots:
image

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <p id="notice"><%= notice %></p>
 
 <p><%= link_to 'Generate Report CSV', posts_path(format: 'csv') %></p>
-
+ <%if user_signed_in?%>
```
Comment (diegosaul402) [inline app/views/posts/index.html.erb]: Please add spaces between the `<%  %>` syntax

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+      <%current_user.avatar.attached?%>
+      <%= image_tag current_user.avatar, :size => "200x200"%>
+    <%end%>
```
Comment (diegosaul402) [inline app/views/posts/index.html.erb]: Same for the lines 5, 6 and 7

File: db/schema.rb [hunk 3]
```diff
@@ -10,14 +10,35 @@
...
+    t.datetime "created_at", null: false
+    t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
+  end
 
   create_table "posts", force: :cascade do |t|
     t.string "title"
     t.text "body"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.references :user, index: true
```
Comment (diegosaul402) [inline db/schema.rb]: Why did you change this?

**Review by diegosaul402 — CHANGES_REQUESTED** [review]
Some small changes, only one doubt about changing back the `user_id` on your posts schema

File: db/schema.rb (hunk 3 above)
Comment (Mildred14) [reply db/schema.rb]: Did I change it? 🤔

Comment (Mildred14) [reply db/schema.rb]: I don't know why I delete that line, in fact, my code has` t.integer : "user_id"`, the old one that I corrected

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -1,9 +1,11 @@
 <p id="notice"><%= notice %></p>
 
 <p><%= link_to 'Generate Report CSV', posts_path(format: 'csv') %></p>
-
+<% if user_signed_in? %>
+  <% current_user.avatar.attached? %>
+  <%= image_tag current_user.avatar, :size => "200x200" %>
```
Comment (8geonirt) [inline app/views/posts/index.html.erb]: Use the new hash syntax 🤓

File: config/environments/development.rb [hunk 5]
```diff
@@ -59,4 +58,7 @@
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
```
Comment (8geonirt) [inline config/environments/development.rb:61]: Is this comment necessary?

**Review by 8geonirt — CHANGES_REQUESTED** [review]
Almost ready! 🤓

**Review by 8geonirt — APPROVED** [review]

**Review by diegosaul402 — APPROVED** [review]
Great job LGTM :shipit:
